### DISCUSSION TITLE
예약 페이지 달력 디자인 제안

### DISCUSSION THREAD

**Discussion by IamJunhee** [chldppwls12/DAYONE_Web#31] [Ideas]
@hjhjhj00 

현재 달력 페이지의 달력은 몇 주인지에 맞춰서 행의 수를 계산합니다.
그래서 5~6주가 있는 달은 캘린더의 행이 늘어나서 비교적 높이가 줄어듭니다.
그래서 다른 캘린더 앱을 찾아봤는데, 보통 행의 수를 일정하게 정해서 높이를 고정하고 대신, 비는 날짜에는 다음 달의 날짜들을 연하게 표시하더라구요!
저희도 그런식으로 구현하면 보기 깔끔하지 않을까 합니다!

[예시 - 구글 캘린더](https://calendar.google.com/calendar/u/0/r) 좌측에 달력을 주목해주세요!

**Comment by hjhjhj00** [comment]
일단 주차만큼 캘린더의 행이 늘어나고, 대신 칸의 간격은 일정하게 하는 걸로 했는데 새로운 디자인 안건이 있어서 이 디자인에 위 예시처럼 한 게 나은 것 같으면 추후에 수정하겠습니다.
